### PR TITLE
issue-425: Documentation for alert extraAnnotations

### PR DESCRIPTION
**Pull request by P6rguVyrst**
I'm more happy with this solution, since it addresses the issue overriding any single alert properties.

Screenshot 2020-05-28 at 20 44 45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +192,41 @@ $ jsonnet -J vendor -S -e 'std.manifestYamlDoc((import "mixin.libsonnet").promet
 $ jsonnet -J vendor -m files/dashboards -e '(import "mixin.libsonnet").grafanaDashboards'
 ```
 
+### Adding custom annotations to alerts
+
+Create a new file: my_custom_mixin.libsonnet with the following:
```
Comment (povilasv) [inline README.md:197]: Can we add a bit more of an overview what we are doing and why?

Comment (P6rguVyrst) [reply README.md:197]: Sure, I'll add more context in the evening.

File: README.md [hunk 2]
```diff
@@ -192,6 +192,41 @@ $ jsonnet -J vendor -S -e 'std.manifestYamlDoc((import "mixin.libsonnet").promet
 $ jsonnet -J vendor -m files/dashboards -e '(import "mixin.libsonnet").grafanaDashboards'
 ```
 
+### Adding custom annotations to alerts
```
Comment (brancz) [inline README.md:195]: I think your intention was to demonstrate how to add annotations to **_specific_** alerts, no?

File: README.md (hunk 1 above)
Comment (P6rguVyrst) [reply README.md]: I added some context to the docs, is it enough? 

You should be able to modify the PR, if you see something you don't like or there's something you see that this section of the docs should include.

File: README.md (hunk 2 above)
Comment (P6rguVyrst) [reply README.md]: Changed this line to be more generic: "Customising alert annotations".

**Pull request merged**